v0/destinations/shynet/data/ShynetPageConfig.json

```diff
@@ -0,0 +1,17 @@
+[
+  {
+    "destKey": "loadTime",
+    "sourceKeys": "context.page.loadTime",
+    "required": false
+  },
+  {
+    "destKey": "referrer",
+    "sourceKeys": "context.page.referrer",
+    "required": false
+  },
+  {
+    "destKey": "location",
+    "sourceKeys": "context.page.url",
+    "required": false
+  }
+]
```

Comment:
Are none of the fields required?

Reply:
nothing mentioned as per doc, made url as required to atleast track the hit on url